### PR TITLE
ci: add GPG settings

### PR DESCRIPTION
**Pull request by javier-godoy**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -505,6 +505,35 @@
 				</plugins>
 			</build>
 		</profile>
+
+		<profiles>
```
Comment (mlopezFC) [inline pom.xml:509]: I think that this tag is wrong

**Review by mlopezFC — CHANGES_REQUESTED** [review]
I think that there is a problem with a tag

**Comment by javier-godoy** [comment]
> I think that there is a problem with a tag

Yes, I had fixed it locally but didn't realize the push didn't went through.

**Review by paodb — APPROVED** [review]

**Review by mlopezFC — APPROVED** [review]
LGTM

**Pull request merged**